colossalai/inference/server/api_server.py

```diff
@@ -202,7 +217,7 @@ def parse_args():
     uvicorn.run(
         app=app,
         host=args.host,
-        port=args.port,
+        port=args.port + 1,
```

Comment:
What's the purpose of adding one here

Reply:
fastapi and colossal ai launch can not use one port

Reply:
You might want to check `free_port` in utils

Reply:
fixed